modules/deploy/pages/deployment-option/self-hosted/manual/linux-system-tuning.adoc

```diff
@@ -0,0 +1,28 @@
+= Linux System Tuning
+:description: Learn how Redpanda applies automatic tunic to your Linux system.
+
+Redpanda includes numerous features to tune your Linux systems for optimal performance. You can trigger hardware-based optimizations through the `rpk redpanda tune` command, also called the autotuner. Software-based configurations are managed automatically through the `systemd` suite of tools.
+
+== User-triggered hardware tuning
+
+Users invoke the Redpanda autotuner using the `rpk redpanda tune` command. This command may optimize both native Linux and Kubernetes nodes. It analyzes the hardware configuration of the worker node and sets appropriate kernel options. You can choose to set certain hardware parameters using `ulimit` or other system tools while using the autotuner to manage others.
```

Comment:
@chrisseto  should vet this statement (see comment below)

Reply:
Yeah I would recommend removing the `and Kubernetes nodes` part. The tuners rely on too many non-Kubernetes friendly (Tuning mounted devices) or Kubernetes inaccessible (Setting up systemd services) things to claim that Kubernetes is well supported or even supported.